modules/tinymce/src/plugins/table/test/ts/browser/TableInsertionTest.ts

```diff
@@ -0,0 +1,39 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/table/Plugin';
+
+import * as TableTestUtils from '../module/test/TableTestUtils';
+
+describe('browser.tinymce.plugins.table.TableCellPropsStyleTest', () => {
+
+  const baseSettings = {
+    plugins: 'table',
+    base_url: '/project/tinymce/js/tinymce'
+  };
+
+  context('TINY-3909: remove_trailing_br is true', () => {
+    const hookWithTrailingBr = TinyHooks.bddSetup<Editor>({ ...baseSettings, remove_trailing_br: true }, [ Plugin ], true);
+
+    it('TINY-3909: remove_trailing_br: true', async () => {
```

Comment:
I am thinking it could be good to test a single table as a test case then rename your existing test case to 'TINY-3909: nested table'. We would have those two test cases for both the 'remove_trailing_br is true' and 'remove_trailing_br is false' context

```suggestion

        it('TINY-3909: single table', async () => {
      const editor = hookWithoutTrailingBr.editor();
      editor.setContent('');
      await TableTestUtils.pInsertTableViaGrid(editor, 2, 2);
      // TODO: assert cursor position - TinyAssertions.assertCursor
      // TODO: assert HTML to make sure brs are present in TDs - TinyAssertions.assertContentPresence
      // TODO: assert returned content - TinyAssertsion.assertContent
    });

    it('TINY-3909: nested table', async () => {
```